security-groups.yaml

```diff
@@ -189,7 +189,6 @@
       protocol: '112'
       remote_ip_prefix: "{{ os_subnet_range }}"
 
-
```

Comment:
These were here to try to differentiate between the different sgs since they're all in the same file, but can be removed if you feel this would be clearer

Reply:
Oh I didn't even notice, my OCD just made me delete them haha. I'm cool with them gone if you are? I was using the search function to find the different SG rules when I was looking.